doc/boot_storage_needed_info.md

```diff
@@ -0,0 +1,38 @@
+List of Information Needed About Storage Configuration
+======================================================
+
+Mount Points
+---------------
+
+- what partition is mounted to "/" ( so it can be queried )
+- what partition is mounted to "/boot" and if it is same as "/" ( so it can be queried )
+- what partition is mounted to "/boot/zipl" ( only on s390 to query )
+- what partition is mounted to "/boot/efi" ( only on efi based archs )
+
+Partitions
+---------------
+
+- is partition logical ( so MBR have to be installed to extended one )
+- extended partition for given disk ( needed only if /boot is on logical partition )
+- file system of partition ( as some fs have problem embedding stage1 code, so do not propose to install it there )
+- fsid of partition ( e.g. to detect bios_grub partition and others )
+- if partition is part of MD, DM or Linux RAID ( target map now returns it, needed to decide if partition can be used for stage1 code )
```

Comment:
I agree that code in this part is quite unclear as it checks both, but when I check storage code, it cannot ever happen. So I will replace requirement by generic term, that we need info about any kind of raid.